mitodl/ocw-hugo-themes
# Added setting button to control bar

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes #140 #139 

#### What's this PR do?
This PR accomplishes the following
- Adds icon button in control bar
- Add playback rate menu and switcher

#### How should this be manually tested?
This PR can be tested by the following method
- Open courses video
- Click icon button
- Verify menu open
- Click playback speed
- Verify playback speed is affected 


#### Any background context you want to provide?
I was looking into why our setPlayBackQuality and why It wasnt working for me while apparently it was for others until i stumbled on this google issue tracker forum https://issuetracker.google.com/issues/123558487
from which I discovered that from  October 2019 setPlayBackQuality is not supported https://developers.google.com/youtube/iframe_api_reference#october-24,-2019

#### Screenshots (if appropriate)

<img width="1348" alt="Screenshot 2022-05-27 at 5 44 38 PM" src="https://user-images.githubusercontent.com/87968618/170705541-63b2eb99-0edf-4cf7-a4e9-08c7df68776c.png">
